package.json

```diff
@@ -25,6 +25,7 @@
   },
   "dependencies": {
     "bluebird": "2.10.1",
+    "clone": "^1.0.2",
```

Comment:
To help avoid conflicts in the future, I suggest omitting the `^` to force a specific version to be installed.